resources/language/resource.language.pt_pt/strings.po

```diff
@@ -301,15 +301,15 @@ msgstr "Reproduzir"
 
 msgctxt "#30140"
 msgid "Adding contents to library"
-msgstr "A adcionar conteúdos na biblioteca"
+msgstr "Adicionando conteúdos à biblioteca"
```

Comment:
Please revert as the previous phrase is more common/used in PT-PT (we usually don't use gerund, specially when starting sentences)